collections/_infographics/emissions/emission-reduction-potential.md

```diff
@@ -3,12 +3,139 @@ layout:        infographic
+
+Planting ten million trees is not unrealistic; in 2019 alone, the Czech Forests planted about 60 million tree seedlings in forests under forest management. Ten million trees would take up about 14,000 hectares (at the density of trees as in an average forest), while about 26,000 hectares of forest have been added in the ten years since the previous forest inventory.
+
+The amount of CO<sub>2</sub> sequestered corresponds to the tree’s wood mass increase. The increase per year corresponds approximately to the tree's size (and therefore age). According to available estimates, a mature tree of 70 years can sequester around 150 kg of CO<sub>2</sub> per year. In 2030, such newly planted trees will be at most 10 years old. Small trees of 10 years of age will thus bind significantly less and our optimistic estimate is 25 kg CO<sub>2</sub> per year. **This gives a total estimate of annual sequestration in 2030 of about 0.25 Mt CO<sub>2</sub>.**
+
+However, this estimate is overestimated for several reasons: first, it assumes that none of the trees planted will die within 10 years. Second, it does not consider what happens to the wood of these trees at the end of their lives - if we burn such lumber, much of the stored carbon will be returned to the atmosphere.
+
+Thus, in terms of the need to substantially reduce emissions by 2030 or 2050, maintaining stable, mature forests would bring higher sequestration, i.e. strictly protecting protected areas and expanding areas without intervention.
+
+Of course, carbon sequestration is not the only criterion. Planting new trees outside forests has many other environmental benefits that justify this project very well, although it does not have much impact on the emission reduction potential.
+
+### Sequestration on arable land (change of management)
+
+Agricultural soils naturally fix carbon and the amount of carbon fixed increases or decreases over the years depending on the farming technique chosen. Over the past decades, long-term studies have been carried out in different parts of the world comparing farming techniques according to their sequestration potential. According to a <!--[-->study<!--](https://aa.ecn.cz/img_upload/410697af7dfcb092dfd4e3937dd69e3f/klima_co2_web_final.pdf)--> by the Food and Agriculture Organisation (part of the United Nations), a hectare of land cultivated under organic farming can store around 200 kg of CO<sub>2</sub> annually. When minimum tillage techniques are added, the sequestration potential increases to about 500 kg of CO<sub>2</sub> per year. On depleted soils, this sequestration can be even higher; long-term studies show that the sequestration potential does not decrease much even after 30 years of such farming.
```

Comment:
Unable to find this link in English. Not sure if there is an equivalent to https://aa.ecn.cz/img_upload/410697af7dfcb092dfd4e3937dd69e3f/klima_co2_web_final.pdf